examples/custom_game_data/game_data.rs

```diff
@@ -74,13 +74,6 @@ impl<'a, 'b> CustomGameDataBuilder<'a, 'b> {
         bundle.build(&mut self.base)?;
         Ok(self)
     }
-    pub fn with_thread_local<S>(mut self, system: S) -> Self
```

Comment:
does this change need to be in? it seems like a separate thing

Reply:
It just isn't used anymore in the example. It was previously because of RenderingSystem, now it was left unused and there was a warning. So I just deleted it.